homeassistant/components/repetier/__init__.py

```diff
@@ -215,6 +215,7 @@ def get_data(self, printer_id, sensor_type, temp_id):
             prop_data = getattr(printer, prop)
             if attr == "temp_data":
                 temp_methods = methods["temp_data"]
+                temp_id = temp_id - 1
```

Comment:
I think this is a bug. In my case (one bed, two extruders) `temp_id` is now -1 and 0 instead of 0 and 1.

Reply:
Yearh - sorry, missed those :) Should be fixed now.

Reply:
Now remove the VS file and we are good to go :smiley:

Reply:
Damn - those VS files :D